src/models/document/document-content-tests/dc-shared-models.test.ts

```diff
@@ -519,29 +531,29 @@ Object {
       expect(getSnapshot(targetDocument)).toMatchInlineSnapshot(`
 Object {
   "rowMap": Object {
-    "testid-57": Object {
+    "testid-69": Object {
       "height": undefined,
-      "id": "testid-57",
+      "id": "testid-69",
       "isSectionHeader": false,
       "sectionId": undefined,
       "tiles": Array [
         Object {
-          "tileId": "testid-55",
+          "tileId": "testid-67",
           "widthPct": undefined,
         },
         Object {
-          "tileId": "testid-56",
+          "tileId": "testid-68",
           "widthPct": undefined,
         },
       ],
     },
   },
   "rowOrder": Array [
-    "testid-57",
+    "testid-69",
   ],
   "sharedModelMap": Object {
-    "sharedDataSet1": Object {
-      "provider": "testid-55",
+    "testid-64": Object {
+      "provider": undefined,
```

Comment:
Is this undefined provider expected?

Reply:
This is kind of expected, because I'm not actually doing anything with `provider`. Right now we have `provider` on `SharedModelEntry` and `providerId` on `SharedDataSet`. As far as I can tell these are redundant, so we should just use one or the other. Maintaining both is confusing and just muddles the code. I don't think `provider` is currently actually doing anything, so I'm updating `providerId` while ignoring `provider`. Am I wrong about any of this?